p5js/00_ImageClassification_MultipleImages/sketch.js

```diff
@@ -0,0 +1,82 @@
+/* ===
+ML5 Example
+00_ImageNet_Simple
+Simple Image Classification using p5.js
+=== */
+
+// Initialize the ImageNet method with the MobileNet model.
+const classifier = new ml5.ImageClassifier('MobileNet');
+
+let img;
+current_index = 0;
+all_images = []
+display = true;
+display_time = 750;
+predictions = []
+
+function append_images(){
+    for (var i in data.images){
```

Comment:
This sort of loop is a little less familiar to beginners. I would suggest a `for` loop.